### DISCUSSION TITLE
Urgent Assistance Needed: Login Issue with GitHub

### DISCUSSION THREAD

**Discussion by Seybo** [vercel/community#5258] [Help] [answered]
### Question

Dear Vercel Support,

I am encountering a critical issue accessing my project due to login problems with my GitHub (GH) account. When attempting to log in, I receive the error: 'There is no Vercel account linked with this GitHub account.' This issue may be related to a recent change in my GH account's email address.

This problem is highly urgent as it is preventing access to my project. I have sent two emails detailing this issue to [registration@vercel.com](mailto:registration@vercel.com) but have not yet received a response. I am concerned about this lack of communication and would greatly appreciate a prompt reply.

Could you please confirm the receipt of this message and provide guidance on how to resolve this login issue? If there is an alternative contact method or additional information required from my side, please let me know.

Thank you for your prompt attention to this matter.

**Comment by Seybo** [comment]
Hi @amyegan , @mcsdevv . Can i get some help here?

**Comment by amyegan** [comment, marked as answer]
Hi @Seybo. I see your ticket was created and assigned to the correct team. Thank you for waiting while the team works through a large number of cases. 

I'm not sure if you tried it already, but logging in with email instead of GitHub is a potential workaround. Please give that a try and let me know if it works for you. 🙏 

https://vercel.com/login/email

**Reply by amyegan** [reply]
I'm transferring this case out of the `vercel` repo and into Vercel Community Support for better visibility.

https://github.com/vercel/vercel/discussions/10973 -> https://github.com/vercel/community/discussions/5258

**Reply by Seybo** [reply]
login with email worked thanks. But i'm pretty sure it didn't work before. Looks like your team (who also contacted me by email yesterday and asked to do the same) update something on your end before that
